nss-day-cohort-39/tabloid-full-stack-velours
# change user profile type

### Changes
- Added a Modal to UserProfile.js that pops up when you click the edit button
- Merged master right before pushing

### Testing
- Confirm that changes from most recent master are working (the terminal did a "recursive" merge)
- Login as an admin and click on "Users" in Header
- Click the edit icon, select a different user profile type, and save
- Make sure list automatically repopulates with the new user profile type appearing
- Click on the "Deactivated Users" button and confirm that the edit icon is not present*
*A user must be activated to edit user profile type
